### PR TITLE
modified subprocess command to work with linux

### PR DESCRIPTION
**Pull request by robertahunt**

### CODE REVIEW

**Review by corradio — COMMENTED** [review]
I think this PR lacks the commit where you fixed the logs right?

File: brick/__main__.py [hunk 1]
```diff
@@ -102,11 +104,11 @@ def docker_build(tags, dockerfile_contents, pass_ssh=False, no_cache=False, secr
                 encoding='utf8',
                 stdout=subprocess.PIPE,
                 stderr=subprocess.PIPE,
-                shell=True,
+                #shell=True,
```
Comment (corradio) [inline brick/__main__.py]: You can simply remove this line. I tested locally and it works!

**Review by corradio — APPROVED** [review]
Thanks!

**Comment by corradio** [comment]
@robertahunt we never merged this. Are you able to solve conflicts and merge?

**Comment by corradio** [comment]
ping @robertahunt

**Comment by robertahunt** [comment]
Yeah, unfortunately the answer is I am not sure. I have been using the master branch of brick for a while now and that seems to be working fine, I am just worried this will come up again. But I can reopen the PR if that happens :/ Sorry about the long wait on this

**Comment by corradio** [comment]
All good! Thanks for the following up

**Pull request closed without merging**